src/Network/Receive/kRO/Sakexe_0.pm

```diff
@@ -3859,4 +3860,52 @@ sub party_dead {
 	}	
 }
 
+sub guild_storage_log {
+	my ($self, $args) = @_;
+	my $msg = $args->{info};
+	my $msg_size = $args->{RAW_MSG_SIZE};
+	my $headerlen = 6;
+	my $guildstotage_pack = 'a4 v V C V a8 C v a8 Z24 Z24 C';
+	my $k = 0;
+	my $guildstotage_len = length pack $guildstotage_pack;
+	my $gstorage;
+	
+	my %gstorage_action = (
+		0 => T('Put'),
+		1 => T('Get'),
+	);
+		
+	if ($args->{result} == 0 || $args->{result} == 1) {
+		my $message = center(T("[ Guild Storage LOG ]"), 40, '-') ."\n".
+			T("# Name				Item		Amount		Action			Time	\n");
+		for (my $i = $headerlen; $i < $msg_size; $i+=$guildstotage_len) {
+		($gstorage->{ID}, #ACCID
```

Comment:
Its not account id. Official Aegis name is "Idx"

Reply:
ty sir .